README.md

```diff
@@ -12,4 +12,48 @@ A small wrapper around redis to allow configuration from confit.
     }
   }
 }
-```
\ No newline at end of file
+```
+
+API
+---
+The redis client attached to `req.gb.db` is an [ioredis](https://github.com/luin/ioredis) Redis object.
+
+Additional Utilities
+--------------------
+
+### Ttl
+Common redis ttl values can be found in `Ttl`.
+```javascript
+req.gb.db.redis.Ttl.oneMinute // => 60
+req.gb.db.redis.Ttl.oneHour // => 3600
+```
+
+### Key Makers
+To generate a key, use `makeKey()`
+```javascript
+req.gb.db.redis.makeKey('redis', 'is', 'cool') // => 'redis:is:cool'
+```
+
+To create a key that begins with the name of your service, use `makeServiceKey()`
+```javascript
+req.gb.db.redis.makeServiceKey('redis', 'is', 'cool') // => 'my-serv:redis:is:cool'
+```
+
+### Caching the Result of a Function Call
+Use `memoize()` to cache the result of a function call.
+
+```javascript
+const redisClient = req.gb.db.redis;
+
+const mySlowDataFunction = async () => {
+  // slow things here
+};
+
+// return cached value. on cache miss, cache and return result of mySlowdataFunction()
+const myVal = redisClient.memoize(
+  mySlowDataFunction,
+  {
+    key: redisClient.makeKey('redis', 'is', 'cool'), // defaults to name of function
```

Comment:
I think I disagree that using the default key is common usage.  In the three services we're calling this function from, we're specifying a key every time.